### PR TITLE
Fix / classic details block create organizer

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Fixes the classic-details-block's organizer creation and, in the process:
- Adds a couple of tests
- Refactors / cleans up the original component a bit
- Fixes the form layout by using WordPress' `TextControls` component instead of a standard input

**Ticket:** https://theeventscalendar.atlassian.net/browse/TEC-3698

**Screenshots:**

Screen Shot 2021-03-12 at 11 30 22 AM

### CODE REVIEW

File: src/modules/elements/organizer-form/element.js [hunk 1]
```diff
@@ -93,29 +97,20 @@ class OrganizerForm extends Component {
 				console.warning( 'Invalid creation of organizer:', newPost );
 			}
 
-			this.props.addOrganizer( newPost );
+			this.props.addOrganizer( newPost.id, newPost );
```
Comment (paulmskim) [inline src/modules/elements/organizer-form/element.js:100]: Did we change the function signature somewhere? I'm not seeing it in this PR.

**Review by paulmskim — APPROVED** [review]
One question, otherwise looks good to me.

File: src/modules/elements/organizer-form/element.js (hunk 1 above)
Comment (alliuca) [reply src/modules/elements/organizer-form/element.js:100]: @paulmskim I don't know why, but it wasn't working at all before. I found out the function requires those arguments

Comment (alliuca) [reply src/modules/elements/organizer-form/element.js:100]: but I'll double check that this same component isn't used by the standard organizer block, maybe that's why

Comment (alliuca) [reply src/modules/elements/organizer-form/element.js:100]: yeah it works everywhere

Comment (paulmskim) [reply src/modules/elements/organizer-form/element.js:100]: hah... well probably why it wasn't working at all

**Pull request merged**